ReactAndroid/src/main/java/com/facebook/react/uimanager/JSTouchDispatcher.java

```diff
@@ -116,14 +116,21 @@ public void handleTouchEvent(MotionEvent ev, EventDispatcher eventDispatcher) {
       mGestureStartTime = TouchEvent.UNSET;
     } else if (action == MotionEvent.ACTION_MOVE) {
       // Update pointer position for current gesture
+      float[] moveTargetCoordinates = new float[2];
```

Comment:
I considered that, but I am not that familiar with this code and its history. I'm not sure if there is other code and/or libraries relying on `mTargetCoordinates` being fixed during a touch. I generally agree that it would make sense to update the coordinates on move and up too. Let me know if I should make that change.